### DISCUSSION TITLE
Oqtane course

### DISCUSSION THREAD

**Discussion by PedroRainho123** [oqtane/oqtane.framework#1288] [General] [answered]
Hi there,

When an oqtane Course including its functionalities and proper custom modules and themes approaches?

Regards.

**Comment by leigh-pointer** [comment, marked as answer]
@PedroRainho123 Hello.
You can take a look at the YouTube video [Oqtane Module Development Series](https://www.youtube.com/playlist?list=PLYhXmd7yV0elLNLfQwZBUlM7ZSMYPTZ_f) This is a great starting point.

I hope this helps.

**Reply by PedroRainho123** [reply]
Dear @leigh-pointer ,

Thanks for your answer. 
Yes, it's a nice starting point. And I already watched it. But that's just it: an introduction of the product. Certainly at some point, related courses will start up. It'd would be nice to have them soon if you're are already diving into it and want to learn the proper and objective ways of doing things.

But let us wait. 😀

Regards.
